inspectit-ocelot-documentation/docs/getting-started/docker-examples.md

```diff
@@ -52,48 +52,37 @@ Sends Zipkin traces to Wavefront through a Wavefront proxy running as a Docker c
-> ```
+:::note 
+Before running the demo, you must add the Wavefront host and API token to the ```.env``` file in the ```inspectit-ocelot-demo```. Please add the following lines and change the cluster and token according to your Wavefront instance.
+```
+WAVEFRONT_URL=https://<your cluster>.wavefront.com/api
+WAVEFRONT_TOKEN=<API token obtained from Wavefront>
+```
+:::
 
 ## Launching the Demo
 
-:::note
+:::warning
 Currently, the demo is only available as Docker images for the AMD64 architecture.
 On ARM platforms there may be problems with the execution.
 :::
 
 *Pre-requisites:* To launch the demo, [Docker](https://www.docker.com/) needs to be installed on your system.
-If you are using Docker for Windows or running Docker in a virtual machine, ensure that Docker has at least 4GB main memory assigned.
+If you are using Docker Desktop or running Docker in a virtual machine, ensure that Docker has at least 4GB main memory assigned.
 
-Either [download](https://github.com/inspectIT/inspectit-ocelot/archive/master.zip) or [clone the inspectit-Ocelot GitHub repository](https://github.com/inspectIT/inspectit-ocelot).
+Either [download](https://github.com/inspectIT/inspectit-ocelot-demo/archive/refs/heads/main.zip) or clone the [inspectIT Ocelot-Demo GitHub repository](https://github.com/inspectIT/inspectit-ocelot-demo).
 
-Change into the ```inspectit-ocelot-demo/``` directory and execute the following command to launch the desired demo scenario (replace [SCENARIO_POSTFIX] with the postfix of the scenario you would like to launch):
+Change into the `spring-petclinic-docker-demo/` directory and execute the following command to launch the desired demo scenario (replace [SCENARIO_POSTFIX] with the postfix of the scenario you would like to launch):
 
 ```bash
 $ docker-compose -f docker-compose-[SCENARIO_POSTFIX].yml up
 ```
 
-This will start all the Docker containers required for the corresponding demo scenario, including the Petclinic demo application.
+This will start all the Docker containers required for the corresponding demo scenario, including the PetClinic demo application.
 
 You can access the demo application (PetClinic) under http://localhost:8080.
 Details on accessing monitoring infrastructure components are listed below, depending on the selected demo scenario.
 
-### Starting the Demo on Windows Using WSL
-
-If you want to execute the demo on Windows using the Windows subsystem for linux (WSL), you have to mount your hard drive to the WSL's root directory due to a problem of Docker for Windows and its volume mounting.
-
-For example, mounting your C drive to the root file system can be achieved using the following commands:
-
-```bash
-sudo mkdir /c
-sudo mount --bind /mnt/c /c
-```
-
-For more information, check out the following blog post: [Setting Up Docker for Windows and WSL to Work Flawlessly](https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly)
-
```

Comment:
As far as my tests went, this is not needed anymore with current versions of Docker Desktop and WSL2, which is why I propose removing it